lessons/pydata/pandas_core/index.ipynb

```diff
@@ -0,0 +1,3402 @@
+{
```

Comment:
<p>Pojmenovávání proměnných. Mohlo by to být třeba <code>pokemoni</code>, <code>tabulka_pokemonu</code> nebo aspoň <code>tabulka</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p><code>tabulka_pokemonu</code> je dobrý nápad</p><p> </p>